{{cookiecutter.project_slug}}/.circleci/config.yml

```diff
@@ -109,8 +109,7 @@ jobs:
           fingerprints:
 {%- endraw %}
             - "{{ cookiecutter.ssh_key_fingerprint }}"
-{% raw -%}
-
+{% raw %}
```

Comment:
Should you update the matching `endraw` too to remove the `-`?

Reply:
No. The other whitespace removal works as anticipated. It removes extra newlines. In this case it removed the newlines and the indentation of `- checkout`.

Reply:
Oh, interesting! Got it then